GSA_Adapter/Convert/ToGsa/Group.cs

```diff
@@ -58,12 +58,25 @@ public static partial class Convert
                 return "\"" + load.Objects.Name + "\"";
 
             //Otherwise apply to the corresponding indecies
-            return load.Objects.Elements.Select(x => int.Parse(x.GSAId().ToString())).GeterateIdString();
+            return load.Objects.Elements.SelectMany(x => x.GSAIds()).OrderBy(x => x).GeterateIdString();
```

Comment:
```suggestion
            return load.Objects.Elements.SelectMany(x => x.GSAIds()).OrderBy(x => x).GenerateIdString();
```

Reply:
As mentioned by @alelom . Prefer to not do any other fixes than more critical bug fixes for the Beta release in this PR. As this has the potential to spill (even if unlikely). I have raised another issue to fix this. https://github.com/BHoM/GSA_Toolkit/issues/260